.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+---
+# See https://pre-commit.com for more information
+# See https://pre-commit.com/hooks.html for more hooks
+default_language_version:
+  python: python3.11
+repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.5.0
+    hooks:
+      - id: mixed-line-ending
+        args: [--fix=lf]
```

Comment:
https://www.aleksandrhovhannisyan.com/blog/crlf-vs-lf-normalizing-line-endings-in-git/#normalizing-line-endings-in-git-with-gitattributes lets you do this with a .gitattributes file too. still not a bad thing to make sure it's done in both spots :D

Reply:
nice - added to the active PR, which is also changing the mixed-line-ending hook